shared/reducers/chat.js

```diff
@@ -17,6 +17,7 @@ function _processMessages (seenMessages: Set<any>, messages: List<Constants.Mess
   const updatedMessages = messages.map(m => messagesToUpdate.has(m.key) ? messagesToUpdate.get(m.key) : m)
   // We have to check for m.messageID being falsey and set.has(undefined) is true!. We shouldn't ever have a zero messageID
   let nextMessages: List<Constants.Message> = filteredPrepend.concat(updatedMessages, filteredAppend).filter(m => !m.messageID || !deletedIDs.has(m.messageID))
```

Comment:
not tyring to change this much but added a flag to tell if we actually did something